src/cf-buttons/usage.md

```diff
@@ -1,11 +1,20 @@
-- [Default button](#default)
-- [Secondary button](#secondary)
-- [Destructive action button](#destructive)
-- [Disabled button](#disabled)
-- [Super button](#super)
+- [Atoms](#atoms)
+  - [Default button](#default-button)
+  - [Secondary button](#secondary-button)
+  - [Destructive action button](#destructive-action-button)
+  - [Disabled button](#disabled-button)
+  - [Super button](#super-button)
+  - [Button links](#button-links)
+  - [Icon buttons](#icon-buttons)
```

Comment:
See the note. I kinda disagree because that change would make all buttons molecules when you look at how the markup is handled.